### DISCUSSION TITLE
Why I cannot access to my Github repositories from HealthCheck

### DISCUSSION THREAD

**Discussion by hamzamaimi** [EddieHubCommunity/HealthCheck#49] [Q&A] [answered]
Hi, I tried to run in local the repo. I followed the readme step by step, so I created my Flagsmith feature then I created a GitHub OAuth app. I added all the stuff in the .env file and started the project but when i try to access to github from the local app by pressing on the Sign in button in the top right I receive this error:
<img width="949" alt="Screenshot 2024-07-22 at 11 08 36 am" src="https://github.com/user-attachments/assets/b5efc325-0ccc-4ffb-85cf-0f1b2a4dd240">
![Screenshot 2024-07-22 at 11 13 56 am](https://github.com/user-attachments/assets/95b2009c-fbb3-4dd3-9583-bc952a191f9a)

**Comment by eddiejaoude** [comment]
Hi @hamzamaimi 

I think there are 2 issues here:
1. the `/repo/list` page I did not push yet, so that 404 is correct for the moment, but I will fix very soon (please ignore for now)
2. yes you need to login to see all the other pages. This sign in error occurs if the GitHub OAuth is not setup properly, can you share your GitHub OAuth page (but please mask/hide any private information)

**Reply by eddiejaoude** [reply]
Here are mine, I will add them to the README also (thank you for raising the question)

<img width="1155" alt="Screenshot 2024-07-22 at 10 49 03" src="https://github.com/user-attachments/assets/cf2e4358-4b13-4c86-a8bb-5c6d83c37b4a">

**Reply by hamzamaimi** [reply]
These are my settings: 
![Screenshot 2024-07-22 at 1 24 01 pm](https://github.com/user-attachments/assets/773ae80a-ab3f-47e7-993b-e39881101e65)
Thanks for your help.

**Reply by eddiejaoude** [reply]
Can you also share your terminal output, I think there will be an error there

**Reply by hamzamaimi** [reply]
Yes you're right, I have a database connection error:

` Can't reach database server at localhost:5432`
How can I create the database? I've already executed the docker compose file

**Reply by eddiejaoude** [reply]
oh interesting, there might be a bug, it should have done the necessary steps, let me check

**Reply by eddiejaoude** [reply]
I think there is a bug, I have raised an issue https://github.com/EddieHubCommunity/HealthCheck/issues/50

**Reply by hamzamaimi** [reply]
Ok, tomorrow morning I'm gonna check and try to figure it out 👍

**Reply by eddiejaoude** [reply, marked as answer]
This should now be fixed in this PR (which I will merge when the GH Actions finish running) https://github.com/EddieHubCommunity/HealthCheck/pull/51
